### PR TITLE
[IMP] pos_customer_display: UI Improvements

### PR DESCRIPTION
**Pull request by vlst-odoo**
Prior to this commit, following recent changes integrating Bootstrap to the customer display, the layout was lacking refinement and required a layout overhaul. This update addresses and improves the UI.

task-3773289

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/point_of_sale/static/src/customer_display/styles.scss
Comment (xlu-odoo) [inline addons/point_of_sale/static/src/customer_display/styles.scss:1]: nit-picking for .o_line_container 
Update to:

```
.o_line_container {
    width: 4rem;
    height: 60px;
    padding: map-get($spacers, 2);

    img {
        max-height: 100%;
        max-width: 100%;
        border-radius: $border-radius-lg !important;
    }
}
```

Will rework the generic component in POS redesign task

**Review by xlu-odoo — CHANGES_REQUESTED** [review]
Thanks for your work, a few comments

**Comment by vlst-odoo** [comment]
robodoo r+

**Comment by caburj** [comment]
robodoo r+

**Comment by caburj** [comment]
robodoo r+

**Pull request closed without merging**